gene/etl/ncbi.py

```diff
@@ -0,0 +1,254 @@
+        info_file = open(self._info_src, 'r')
+        info = csv.reader(info_file, delimiter='\t')
+        next(info)
+
+        with self._database.genes.batch_writer() as batch:
+            for row in info:
+                is_valid_row = True
+                params = {
+                    'concept_id': f"ncbigene:{row[1]}",
+                }
+                # get symbol
+                if row[2] != '-':
+                    params['symbol'] = row[2]
+                else:
+                    logger.error(f"Couldn't read symbol from row: {row}")
+                # get aliases
+                if row[4] != '-':
+                    params['aliases'] = row[4].split('|')
+                else:
+                    params['aliases'] = []
+                # get other identifiers
+                if row[5] != '-':
+                    xrefs = row[5].split('|')
+                    other_ids = []
+                    for ref in xrefs:
+                        if ref.startswith("HGNC:"):
+                            prefix = NamespacePrefix.HGNC.value
+                        elif ref.startswith("MIM:"):
+                            prefix = NamespacePrefix.OMIM.value
+                        elif ref.startswith("IMGT/GENE-DB:"):
+                            prefix = NamespacePrefix.IMGT_GENE_DB.value
+                        else:
+                            prefix = ref.split(':')[0].lower()
+                            if prefix not in VALID_CID_PREFIXES:
+                                logger.error(f"invalid ref prefix {prefix}"
+                                             f" in:\n {row}")
+                                is_valid_row = False
+                                break
+                        other_id = f"{prefix}:{ref.split(':')[-1]}"
+                        other_ids.append(other_id)
+                    params['other_identifiers'] = other_ids
+                else:
+                    params['other_identifiers'] = []
+                # get label
+                if row[8] != '-':
+                    params['label'] = row[8]
+                # add prev symbols
+                if row[1] in prev_symbols.keys():
+                    params['previous_symbols'] = prev_symbols[row[1]]
+                if is_valid_row:
+                    self._load_data(Gene(**params), batch)
+        info_file.close()
+
+    def _load_data(self, gene: Gene, batch):
+        """Load individual Gene item.
+
+        :param Gene gene: gene instance to load into db
+        :param batch: boto3 batch writer
+        """
+        item = gene.dict()
+        concept_id_lower = item['concept_id'].lower()
+
+        if item['symbol']:
+            pk = f"{item['symbol'].lower()}##symbol"
+            batch.put_item(Item={
+                'label_and_type': pk,
+                'concept_id': concept_id_lower,
+                'src_name': SourceName.NCBI.value
+            })
+        else:
+            del item['symbol']
```

Comment:
We can remove this check
```suggestion
        pk = f"{item['symbol'].lower()}##symbol"
        batch.put_item(Item={
            'label_and_type': pk,
            'concept_id': concept_id_lower,
            'src_name': SourceName.NCBI.value
        })
```